src/views/AddItem.jsx

```diff
@@ -20,8 +21,23 @@ export function AddItem({ listToken }) {
 		setItem({ ...item, [e.target.name]: updateVal });
 	};
 
+	const isInvalid = (name) => {
```

Comment:
Great decision to write this in AddItem, while importing the isEmpty and isDuplicate functions - keeping logic and intent/use close to the component, while breaking out the reusable parts.

Reply:
Wanted to keep all the state changes inside components!